packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/runtimeIcon.st

```diff
@@ -0,0 +1,8 @@
+icons
+runtimeIcon
+
+	^ Form
```

Comment:
Könntet ihr, wenn das nicht zu lang ist (dann machen wir das in Persona), kurz erklären wie das hier funktioniert? Ich nehme an es generiert eine Form mit 12x12px, der Rest ist leider nicht so deskriptiv? :/

Reply:
Klaro! :) Grundsätzlich initialisiert das eine neue `Form`-Instanz mit 32 bit Farbtiefe und einer Größe von 12x12 Pixel. Der Parameter `formArray` gibt einfach die einzelnen Werte für die insgesamt 12x12 = 144 Pixel an.

Das scheint in Squeak die übliche Methode zu sein, Icons zu nutzen, sind ja schließlich nur sehr kleine Dateien (s. z. B. die `ToolIcons`-Klasse, die z. B. vom Test-Runner verwendet wird und aus der wir dieses Icon auch stibitzt haben). Auf jeden Fall können die Icons so einfach versioniert werden und müssen nicht manuell z. B. im `Ressources`-Ordner abgelegt werden.

Wir haben das Icon in diesem Fall kopiert (und nicht direkt aus der `ToolIcons`-Klasse referenziert), um es semantisch benennen und nutzen können. So wird z. B. auch verhindert, dass das Icon im Profiler durch ein unpassendes ersetzt wird, nur weil sich das Icon für den Test Runner ändert. (Außerdem ist die `ToolIcons`-Klasse nicht in den Standard-Squeak-Images enthalten, weshalb die Tests failen würden.)

Reply:
Super, klingt nach einer eleganten Lösung! Danke.